backend/rdf4j/src/main/java/org/dotwebstack/framework/backend/rdf4j/query/StaticQueryFetcher.java

```diff
@@ -55,10 +77,48 @@ public Object get(@NonNull DataFetchingEnvironment environment) {
         .orElseGet(TreeModel::new);
   }
 
+  private Object getQueryResultAsModel(@NonNull DataFetchingEnvironment environment,
```

Comment:
Model teruggeven gezien de method naam?
```suggestion
  private Model getQueryResultAsModel(@NonNull DataFetchingEnvironment environment,
```